src/otx/core/exporter/native.py

```diff
@@ -118,7 +115,8 @@ def to_onnx(
         Returns:
             Path: The path to the saved ONNX model.
         """
-        dummy_tensor = torch.rand(self.input_size).to(next(model.parameters()).device)
+        input_size = (1, 3, *self.data_input_params.input_size)
```

Comment:
I'd opt to move this operation out from the exporter, as it becomes not that generic. Once can add a method to the input params to generate `NCHW` shape for instance

Reply:
I added a method to DataInputParams to return input_size as NCHW format.